src/main.rs

```diff
@@ -21,7 +22,8 @@ use crate::{
 // Re-export derive macros.
 use retis_derive::*;
 
-const VERSION_NAME: &str = "bredele";
+static VERSION_NAME: Lazy<&str> =
+    Lazy::new(|| option_env!("RETIS_RELEASE_NAME").unwrap_or("unreleased"));
```

Comment:
Sorry, didn't see this before. Shouldn't this be defined alongside the version in `cli.rs`? It was defined here historically as it seemed a better place for something we had to modify for each release, but thanks to your work it's now defined in Cargo.toml so we can safely move it. That would also save a once_cell use.

nit: I think it's safe to keep the version & name mandatory (another comment suggested to always set those variables).

Reply:
yeah, among the many things to take care of in this PR I miss this. That was a good catch, though. It's time to remove this, indeed (not super neat, but the reasoning is/was clear).

Reply:
I pushed an updated version keeping things as it was, but let's not consider this resolved until we reach consensus :)

Reply:
> Jokes aside the reason of option_env!() over env!() is practical and it's about development. The split build (make ebpf; cargo build) is simply more convenient instead of being forced to specify both env variables.

Makes sense, the current status is good.